src/Command/DrupalUpdate/CheckPackageInfo.php

```diff
@@ -0,0 +1,210 @@
+<?php
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+use Composer\Semver\Comparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CheckPackageInfo
+{
+  /**
+   * @var array
+   */
+  public array $infoPackageFiles = [];
+
+  /**
+   * @var array
+   */
+  public array $infoDetailPackages = [];
+
+  /**
+   * @var string[]
+   */
+  public array $packageInfoKey = [
```

Comment:
As far as I can tell, this is used only once as a local variable. Does it need to be a property?